.pre-commit-config.yaml

```diff
@@ -45,4 +45,6 @@ repos:
   - id: add-license-headers
     files: '(src|examples|tests)/(?:.*/)*[^./]*(\.py)?$'
     exclude : ^src/ansys/mechanical/env/mechanical-env$
+    args:
+    - --start_year=2023
```

Comment:
what does this mean?

Reply:
this is for adding license headers, since mechanical-env does not have extension, ansys pre-commit hook shows error. Also, excluding test.